apps/api/src/utils/resume_handler.py

```diff
@@ -11,20 +11,29 @@
 
 log = getLogger(__name__)
 
-RESUMES_FOLDER_ID = os.getenv("RESUMES_FOLDER_ID")
+HACKER_RESUMES_FOLDER_ID = os.getenv("HACKER_RESUMES_FOLDER_ID")
+MENTOR_RESUMES_FOLDER_ID = os.getenv("MENTOR_RESUMES_FOLDER_ID")
```

Comment:
Do you think you could make a similar update to #465 when you have the chance, thanks.